CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/) and this
 
 ### Added
 
+- `maxReachedTooltip` prop for the `CheckboxGroup` and `CheckboxChipGroup` component.
```

Comment:
```suggestion
- `maxReachedTooltip` prop for the `CheckboxGroup` and `CheckboxChipGroup` components.
```

Reply:
And maybe a few words about what it does, like _"... to customize the hover text when `max` items have been selected"_